app/helpers/buyers_helpers.py

```diff
@@ -59,6 +59,30 @@ def add_unanswered_counts_to_briefs(briefs):
     return briefs
 
 
+def add_response_counts_to_briefs(briefs, data_api_client):
+    for brief in briefs:
+        responses = data_api_client.find_brief_responses(brief_id=brief['id'])
+        brief['responses_count'] = len(responses)
+
+    return briefs
+
+
 def clarification_questions_open(brief):
     # TODO: Implement this properly
     return True
+
+
+def classify_and_count_brief_responses(brief_id, data_api_client):
+    brief_responses = data_api_client.find_brief_responses(brief_id)['briefResponses']
+    failed_count = 0
+    eligible_count = 0
+    for brief_response in brief_responses:
+        if all_essentials_are_true(brief_response):
+            eligible_count += 1
+        else:
+            failed_count += 1
+    return failed_count, eligible_count
```

Comment:
I would be happier if this function was named something that gave more of a clue about what's being returned.  As it's just two integers, it's possible someone could mix them up.

`return_counts_for_failed_and_eligible_brief_responses` maybe?